src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -191,36 +187,37 @@ void EnRz_ActorShadowFunc(Actor* thisx, Lights* mapper, PlayState* play) {
 }
 
 void EnRz_ChangeAnim(PlayState* play, EnRz* this, s16 animIndex, u8 animMode, f32 transitionRate) {
-    static AnimationHeader* D_80BFCD20[] = {
-        &object_rz_Anim_00457C, &object_rz_Anim_003A20, &object_rz_Anim_005E50, &object_rz_Anim_003098,
-        &object_rz_Anim_00059C, &object_rz_Anim_000DE8, &object_rz_Anim_0028D4,
+    static AnimationHeader* sJudoAnimations[] = {
+        &gRosaSistersThinkingAnim, &gRosaSistersStandingAnim,   &gRosaSistersWalkingWhileThinkingAnim,
+        &gRosaSistersSittingAnim,  &gRosaSistersApplaudingAnim, &gRosaSistersOnKneesAnim,
+        &gRosaSistersDancingAnim,
     };
-
-    static AnimationHeader* D_80BFCD3C[] = {
-        &object_rz_Anim_003A20, &object_rz_Anim_003A20, &object_rz_Anim_005390, &object_rz_Anim_003098,
-        &object_rz_Anim_00059C, &object_rz_Anim_000DE8, &object_rz_Anim_0028D4,
+    static AnimationHeader* sMarillaAnimations[] = {
```

Comment:
I'd note that this has to be the same size as `sJudoAnimations` somewhere. The alternative is to just use `EN_RZ_ANIM_MAX` everywhere instead of ARRAY_COUNT, and do
```suggestion
    static AnimationHeader* sMarillaAnimations[EN_RZ_ANIM_MAX] = {
```

Reply:
i'm failing to find a match with this, because sJudoAnimations and sMarillaAnimations are size 7 and EN_RZ_ANIM_MAX equals 9

Reply:
Just leave a note instead then, the person who finishes documenting this can sort it out.

Reply:
What should the comment say?

Reply:
`// N.B. both of these arrays must be the same length due to how the animations are chosen` would do.

Reply:
Aren't they the same size already?

Reply:
Yes, but the point is if you add something to one you have to add something to the other.